app/pods/publications/template.hbs

```diff
@@ -24,7 +24,7 @@
                                       @placeholder={{t "search-placeholder"}}
                                       @value={{this.searchText}}
                                       {{on "input" (perform this.debouncedSearchTask)}}
-                                      @enter={{perform this.debouncedSearchTask}}
+                                      {{!-- @enter={{perform this.debouncedSearchTask}} --}}
```

Comment:
Enter is hier niet nodig, die on input pikt alles op.
Als enter toch nodig is, dan heb je geen "event.target.value"
dan is "event" de tekst dat getypt is

Reply:
> Enter is hier niet nodig, die on input pikt alles op.

Ongeveer, maar voor snelle typers kan je met "enter" meteen de zoekopdracht starten, zonder de debounce timeout af te wachten. Verscheidene verbeteringen hieromtrent zitten ondertussen op `development`.